test/linalg/svd.jl

```diff
@@ -4,6 +4,31 @@ using Base.Test
 
 using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted
 
+@testset "Simple svdvals / svdfact tests" begin
+    ≈(x,y) = isapprox(x,y,rtol=1e-15)
```

Comment:
probably better to use the kwarg capability of at-test for this, or an operator that doesn't already have a meaning